drivers/sensor/openrocket/openrocket_sensors.c

```diff
@@ -8,7 +8,7 @@
 #include <zephyr/random/random.h>
 #endif
 
-LOG_MODULE_REGISTER(openrocket, CONFIG_OPENROCKET_LOG_LEVEL);
+LOG_MODULE_REGISTER(openrocket, LOG_LEVEL_INF);
```

Comment:
The openrocket logging solution. Its that easy

And now we get timestamps which will let us CI check "detect boost within 0.5s of occurence"